apps/miiverse-api/src/services/internal/server.ts

```diff
@@ -75,12 +78,12 @@ export async function setupGrpc(): Promise<void> {
 				status: result.status,
 				payload: JSON.stringify(result.body)
 			};
-		},
-		sMMRequestPostId: async (_request) => {
-			throw new ServerError(Status.UNIMPLEMENTED, 'Not implemented');
 		}
 	});
 
+	// External communication (other pretendo services --> miiverse-api)
+	server.with(apiKeyMiddleware).add(MiiverseServiceDefinition, miiverseDefinition);
```

Comment:
A nice improvement for the future would be to have a different (or 2nd-order function or whatever) API key check here, so that there can be unique keys for the various services.
Not really relevant right now but might be interesting if we ever want to open up the "internal" api for third-party clients